runtime/compiler/control/CompilationThread.cpp

```diff
@@ -12973,7 +12973,10 @@ TR::CompilationInfo::addOutOfProcessMethodToBeCompiled(JITServer::ServerStream *
       {
       // Initialize the entry with defaults (some, like methodDetails, are bogus)
       TR::IlGeneratorMethodDetails details;
-      entry->initialize(details, NULL, CP_SYNC_NORMAL, NULL);
+      // stream == 0 is a special case of request that performs AOTCache loads from disk
```

Comment:
Need to use `LOAD_AOTCACHE_REQUEST` here, in both the code and the comment.

Reply:
Good find. Fixed